sdcm/utils/common.py

```diff
@@ -985,6 +985,26 @@ def tag_ami(ami_id, tags_dict, region_name):
     LOGGER.info("tagged %s with %s", ami_id, tags)
 
 
+def get_tables_with_compaction(tester, db_node, ks, compaction=True):
```

Comment:
[compaction](https://docs.scylladb.com/kb/compaction/) has different meaning in Scylla.
move this to tester class (in future we will move those methods to test_lib/db.py)
API should be
```
get_db_tables(session, key_space=None, with_compact_storage=False)
```
also need to remove `_get_all_non_compact_tables` since this method will return the result